tests/test_capabilities.py

```diff
@@ -174,6 +176,37 @@ def test_subscope_bb_rules(z9324d_extractor):
     assert "test rule" in capabilities
 
 
+def test_match_specific_functions(z9324d_extractor):
+    rules = capa.rules.RuleSet(
+        [
+            capa.rules.Rule.from_yaml(
+                textwrap.dedent(
+                    """
+                    rule:
+                        meta:
+                            name: receive data
+                            scopes:
+                                static: function
+                                dynamic: call
+                            examples:
+                            - 9324d1a8ae37a36ae560c37448c9705a:0x401CD0
+                        features:
+                            - or:
+                                - api: recv
+                    """
+                )
+            )
+        ]
+    )
+    extractor = FunctionFilter(copy(z9324d_extractor), {0x4019C0})
```

Comment:
Ah interesting. This is maybe a good reason not to mutate the extractor directly and wrap it instead.

Reply:
Oh good point... Hmm I think I'll try to implement that instead.

Reply:
thanks. that may have its own issues - lets see.